.github/workflows/ci.yml

```diff
@@ -9,14 +9,18 @@ jobs:
     - uses: actions/checkout@v2
     - name: setup
       run: |
-        rustup install stable-2020-06-18-x86_64-unknown-linux-gnu
-        rustup component add rustfmt --toolchain stable-2020-06-18-x86_64-unknown-linux-gnu
-        rustup component add clippy --toolchain stable-2020-06-18-x86_64-unknown-linux-gnu
+        rustup install 1.51.0
+        rustup component add rustfmt --toolchain 1.51.0
+        rustup component add clippy --toolchain 1.51.0
     - name: style
       run: cargo fmt -- --check
     - name: clippy
-      run: cargo clippy --all-targets --all-features -- -D warnings
-  
+      run: |
+        cargo clippy --all-targets --features rayon -- -D warnings
```

Comment:
Why switch from --all-features to manually specifying all 4 features?

Reply:
it will report a false error regarding to feature switch:
```
cargo clippy --all-targets --all-features -- -D warnings

error[E0432]: unresolved import `crate::lock`
 --> src/set.rs:3:12
  |
3 | use crate::lock::RwLock;
  |            ^^^^ maybe a missing crate `lock`?

...
```